playbooks/roles/logger/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+logger_address: 192.168.9.90
```

Comment:
Probably default all server names to 127.0.0.1 and then we can override in `vagrant-elk.yml`.